brainreg/backend/niftyreg/niftyreg_binaries.py

```diff
@@ -9,17 +11,36 @@
     os_folder_name = __os_folder_names[platform.system()]
 except KeyError:
     raise ValueError(
-        "Platform {} is not recognised as a valid platform. "
-        "Valid platforms are : {}".format(
-            platform.system(), __os_folder_names.keys()
-        )
+        f"Platform {platform.system()} is not recognised as a valid platform. "
+        f"Valid platforms are : {__os_folder_names.keys()}"
     )
 
 
-def get_binary(binaries_folder, program_name):
-    path = os.path.join(binaries_folder, os_folder_name, program_name)
-    return path
+def conda_install_path() -> Optional[Path]:
+    """
+    If a conda install of niftyreg is available, return the directory
+    containing the niftyreg binaries.
+    """
+    if "CONDA_PREFIX" not in os.environ:
+        return
 
+    bin_path = Path(os.environ["CONDA_PREFIX"]) / "bin"
+    if (bin_path / "reg_aladin").exists():
+        return bin_path
 
-def get_niftyreg_binaries():
-    return resource_filename("brainreg", "bin/nifty_reg")
+
+_CONDA_INSTALL_PATH = conda_install_path()
+
+
+def get_binary(program_name: str) -> Path:
+    """
+    Get path to one of the niftyreg binaries.
+
+    If niftyreg is installed via. conda uses those binaries, otherwise falls
+    back on bundled binaries.
+    """
```

Comment:
```suggestion
    If niftyreg is installed via conda, use those binaries, otherwise fall
    back on bundled binaries.
    """
```
or something like this?